src/styles/block/header-style.scss

```diff
@@ -0,0 +1,87 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+}
+
+body {
+  padding: 0;
+  margin: 0 auto;
```

Comment:
The `body` rule should have a max-width of 1200px to follow the Code Style Rules.